pytrickle/frames.py

```diff
@@ -179,4 +244,235 @@ def with_monotonic_timestamps(cls, frames: List[AudioFrame], request_id: str, st
             corrected_frame = AudioFrame._from_existing_with_timestamp(frame, corrected_timestamp)
             corrected_frames.append(corrected_frame)
         
-        return cls(corrected_frames, request_id) 
\ No newline at end of file
+        return cls(corrected_frames, request_id)
+
+
+# Frame Processing Utilities
+# ===========================
+
+class FrameProcessor:
+    """Reusable frame conversion and processing utilities."""
+    
+    @staticmethod
+    def convert_trickle_to_av(frame: VideoFrame) -> av.VideoFrame:
+        """Convert pytrickle VideoFrame to av.VideoFrame."""
+        return tensor_to_av_frame(frame.tensor)
+    
+    @staticmethod
+    def convert_av_to_trickle(av_frame: av.VideoFrame, original_frame: VideoFrame) -> VideoFrame:
+        """Convert av.VideoFrame back to pytrickle VideoFrame with original timing."""
+        # Convert av frame tensor back to normalized float tensor
+        frame_np = av_frame.to_ndarray(format="rgb24").astype(np.float32) / 255.0
+        tensor = torch.from_numpy(frame_np)
+        
+        return VideoFrame(
+            tensor=tensor,
+            timestamp=original_frame.timestamp,
+            time_base=original_frame.time_base
+        )
+
+    @staticmethod
+    def create_processed_frame(processed_tensor: torch.Tensor, original_frame: VideoFrame) -> VideoFrame:
```

Comment:
same as above
```suggestion
    def create_processed_video_frame(processed_tensor: torch.Tensor, original_frame: VideoFrame) -> VideoFrame:
```

Reply:
Fixed in https://github.com/livepeer/pytrickle/commit/70a0ede3859a13661aa4f86560500e13dc8a4f89 (see above)